src/NuGet.Core/NuGet.Protocol/Plugins/OutboundRequestContext`1.cs

```diff
@@ -21,6 +21,7 @@ public sealed class OutboundRequestContext<TResult> : OutboundRequestContext
         private bool _isClosed;
         private bool _isDisposed;
         private bool _isKeepAlive;
+        private readonly IPluginLogger _logger;
```

Comment:
This is not a feedback on this declaration but rather that the file name is weird. 

`OutboundRequestContext`1.cs`

I'd imagine it needs to be `OutboundRequestContext.cs`. Doesn't need to change in this PR ofc.

Reply:
There is already an [OutboundRequestContext.cs](https://github.com/NuGet/NuGet.Client/blob/1fe979bfbfcd1debf492d27446c3474c42a58c5b/src/NuGet.Core/NuGet.Protocol/Plugins/OutboundRequestContext.cs) for the non-generic `OutboundRequestContext` class.  This file is for the generic subclass `OutboundRequestContext<TResult>`.